addons/survey/static/src/scss/survey_views.scss

```diff
@@ -5,13 +5,32 @@
 }
 
 .o_kanban_card_survey_successed{
+    background-image:
+        -webkit-linear-gradient(rgba(255,255,255,.9), 
+                        rgba(255,255,255,.9)),
+        url(/survey/static/src/img/trophy-solid.svg);
+    background-image:
+        -moz-linear-gradient(rgba(255,255,255,.9), 
+                        rgba(255,255,255,.9)),
+        url(/survey/static/src/img/trophy-solid.svg);
+    background-image:
+        -ms-linear-gradient(rgba(255,255,255,.9), 
+                        rgba(255,255,255,.9)),
+        url(/survey/static/src/img/trophy-solid.svg);
+    background-image:
+        -o-linear-gradient(rgba(255,255,255,.9), 
+                        rgba(255,255,255,.9)),
+        url(/survey/static/src/img/trophy-solid.svg);
     background-image:
         linear-gradient(rgba(255,255,255,.9), 
                         rgba(255,255,255,.9)),
         url(/survey/static/src/img/trophy-solid.svg);
     background-repeat: no-repeat;
-    background-position: bottom 6px left -45px;
-    background-size: 100px;
+    background-position: bottom 0 left -45px;
+    -webkit-background-size: 100px;
+    -moz-background-size: 100px;
+    -o-background-size: 100px;
+    background-size: 100px 100px;
```

Comment:
- You should only add the vendor prefix for the browsers which do not support the property naturally
    - I doubt background-size is not supported on any browser (https://caniuse.com/mdn-css_properties_background-size clearly no need for vendor prefix there)
    - linear gradient seems really supported too now, a visual bug on safari for windows...? I really would ignore it. Also are sure this actually solves the bug? Like you said the last version on windows is 5.1 but support with prefix is only added from 6.1 according to caniuse (https://caniuse.com/?search=linear-gradient)
- If one is actually needed: this code should be changed instead of the SCSS: https://github.com/odoo/odoo/blob/12.0/odoo/addons/base/models/assetsbundle.py#L521

cc @sbuhl